### ISSUE TITLE
Achievement category selection drop down menu stuck on screen

### ISSUE THREAD

**Issue by echaskaris** [ValveSoftware/steam-for-linux#7825]
#### Your system information

* Steam client version (build number or date): 1622157314
* Distribution (e.g. Ubuntu): Ubuntu 20.04
* Opted into Steam client beta?: [Yes/No] y
* Have you checked for system updates?: [Yes/No] y
* 
#### Steps for reproducing this issue:

1. go to the achievements page for a game eg TF2
2. open the achievements category menu and select eg general achievements
3. the drop down menu is stuck on screen. To remove it you have to restart steam.

I'm using X11.

**Comment by echaskaris** [comment]
I tested on Wayland and the bug occurs then as well.

**Comment by HanFox** [comment]
This happens on most dropdowns, not just achievements.

Similar report on the Steam beta Discussions board: https://steamcommunity.com/groups/SteamClientBeta/discussions/0/3106901665842520541/

**Comment by ehea617** [comment]
I am also experiencing this. As @HanFox mentioned, it appears to happen with most dropdowns that are displayed inside Steam's web browser window. This issue is very annoying because I have to completely close and reopen Steam to make the dropdown go away.

#### Your system information

* Steam client version (build number or date): 1624329075
* Distribution (e.g. Ubuntu): Debian Testing
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
Desktop environment: Cinnamon Desktop on X11. Any dropdown menus rendered inside Steam's browser window do not close and get stuck on the screen, requiring a client restart to go away.

#### Steps for reproducing this issue:

1. Click on a dropdown menu inside the Steam browser window. Ex. [age check dropdowns](https://store.steampowered.com/agecheck/app/623280/)
2. The dropdown menu will become "stuck" on the screen
3. The only way for the dropdown to go away is to restart the Steam client

**Comment by gardockt** [comment]
Also happens on Arch Linux with i3-gaps, however I have found two workarounds that make the dropdown menu disappear:
- Hover the cursor over one of the options above browser window (Store/Library/Community/[nickname]) - does not seem to be working consistently,
- Switch between tiling/floating mode - this, however, makes the webpage freeze, so a refresh is needed.

**Comment by chknatwork** [comment]
Can confirm that this happens in Mankato, too and I am using the Steam Runtime client.

**Comment by SkyTheGreatOne** [comment]
This also happens on all drop down menus in Linux Mint. Also noticed that resizing Steam's window from the top can cause the stuck drop down menu to disappear, however the page does not function after and needs to be refreshed.

**Comment by kisak-valve** [comment]
There's too many separate examples of drop down menus getting stuck on screen in libcef rendered sections for this to be a one-off issue. Since this is the first report of the issue, let's use this issue report to track it.

Reported examples:
#7864 Age confirmation
#7931 Searching discussions
#7943 Drop down menus in the marketplace

**Comment by Zagnaut1717** [comment]
Currently experiencing this on Pop! OS 20.04 and 21.04.

**Comment by dimitris-triantafyllidis** [comment]
Just noticed this in the age verification page, on Arch Linux and with all the window managers I have installed, namely i3, Openbox and bspwm.

**Comment by Ruedii** [comment]
This issue is likely upstream in CEF

SO many bugs come from CEF.  I don't know why they don't switch to qtWebKit, KHTML or GTK-WebKit instead.  All three of these are also more lightweight than CEF, and allow attaching your choice of javascript interpreters so you can choose one nice and light and tune the GC not to interfere with gameplay.

**Comment by aalouaoui** [comment]
Same issue for me on all drop down menus on Steam.

**Comment by mseancole** [comment]
Experienced this on both PopOS 20.04 and Ubuntu 20.04, mainly via the age confirmation. However, the dropdowns at checkout for payment information don't seem to have the same problem.

**Comment by jarst** [comment]
Confirmed. System information:
  Steam client version (build number or date): 1626824053
  Distribution (e.g. Ubuntu): Ubuntu 20.04.3
  Opted into Steam client beta?: No
  Have you checked for system updates?:  Yes
  NVIDIA GM107M [GeForce GTX 960M] driver: nvidia v: 460.91.03

Drop down stuck on screen even after Steam client was closed.
In order to make stuck drop down disappear:
1. Change size of Steam client window so that it no longer takes whole screen,
2. Move Steam client window.

**Comment by smstewa4** [comment]
I am also experiencing this issue. Dragging the Steam window around a bit does work, as does completely quitting Steam.

Distro: Arch Linux
Steam package versions: 1626824053
Graphics: NVIDIA 2070 Super on driver version 470.63.01-4
Kernel Version: 5.13.13.arch1-1
Window Manager: GNOME 40 on X11

**Comment by Merely1Man** [comment]
Plasma user, same problem. I'll keep the previously posted workarounds in mind, but it remains a nuisance.

Distro: KDE Plasma 5.22
Graphics: AMD Radeon RX 480

**Comment by atirut-w** [comment]
Ubuntu 21.04 here, very annoying to have to restart Steam for the stuck menu to go away.

**Comment by eNTi** [comment]
Artix Linux here. Same issue. Even during payment. The drop down just overlays everything... can't even pay. This is a very old issue but it seems to be getting worse.

**Comment by drsparklegasm** [comment]
Yup, also using Manjaro, but I believe I've noticed this in any KDE interface. Not sure if it happened on ubuntu gnome or not

**Comment by vaslabs** [comment]
this happens for me too, fedora 34

**Comment by n3-ss** [comment]
Happening for me on Pop!_OS 21.04 running Gnome, have been experiencing this for a few months now at least.

**Comment by Cytomax55** [comment]
I have it on Manjaro Plasma stable and beta
OS: Manjaro 21.1.6 Pahvo
Kernel: x86_64 Linux 5.14.10-1-MANJARO
Uptime: 3h 35m
Packages: 1624
Shell: bash 5.1.8
Resolution: 1920x1200
DE: KDE 5.87.0 / Plasma 5.22.5
WM: KWin
GTK Theme: Breath [GTK2/3]
Icon Theme: breath2
Disk: 9.3T / 115T (9%)
CPU: AMD Ryzen 7 3800X 8-Core @ 16x 3.9GHz
GPU: AMD Radeon RX 5700 XT (NAVI10, DRM 3.42.0, 5.14.10-1-MANJARO, LLVM 12.0.1)
RAM: 9714MiB / 15943MiB

**Comment by atirut-w** [comment]
> just resize the window

oh my god you're a savior

**Comment by Brad7173** [comment]
Same issue here in Kubuntu current version. Switching to Beta Steam participation is not an option because of issue #8226. The joysticks are on an endless cycle of being detected then not detected which is also causing Kubuntu problems.

**Comment by atirut-w** [comment]
> which is also causing Kubuntu problems

Doesn't sound like a Steam problem

**Comment by Brad7173** [comment]
> > which is also causing Kubuntu problems
> 
> Doesn't sound like a Steam problem

Yes it's a steam issue. This did not occur until the Nov 11 update.

**Comment by atirut-w** [comment]
Steam issues shouldn't be affecting your whole OS like that. I'm not experienced in this field, though, so take my words with a huge grain of salt.

**Comment by Brad7173** [comment]
I've used Kubuntu for the last 10-15 years. So I know for a fact it is in fact a steam issue.
